client/app/scripts/main.dev.js

```diff
@@ -1,18 +1,18 @@
-require('font-awesome-webpack');
-require('../styles/main.less');
-require('../images/favicon.ico');
-
 import 'babel-polyfill';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
-import configureStore from './stores/configureStore.dev';
-
-import DevTools from './components/dev-tools';
 import Immutable from 'immutable';
 import installDevTools from 'immutable-devtools';
-installDevTools(Immutable);
 
+import configureStore from './stores/configureStore.dev';
+import DevTools from './components/dev-tools';
+
+require('font-awesome-webpack');
+require('../styles/main.less');
+require('../images/favicon.ico');
```

Comment:
Right, I replaced every `require(X);` with `import X;` in `main.js`, `main.dev.js`, `contrast-main.js` and `terminal-main.js`.